docs/readmewidget.md

```diff
@@ -89,7 +89,7 @@ The navigation context manages the screens stack and transitions and exposes fun
 Exports `NavContext`, `NavProvider` and `NavContainer`.
 
 `NavProvider`: Navigation provider. Wraps the components that will consume the context.
-`NavContainer`: Container of the screens. The component that will be used as main screen should be passed as prop to this component. [See implementation here](./index.tsx).  
+`NavContainer`: Container of the screens. The component that will be used as main screen should be passed as prop to this component. [See implementation here](https://docs.onramper.dev/widget/#react-component).  
```

Comment:
Shouldn't this point to the code on github?

Reply:
Perhaps // probably

Reply:
Right, and this .md should be on the widget repo, I'll move it now and fix the links